tests/eval_tests/utils.py

```
@@ -173,25 +225,60 @@ def subplots_comparison(df, column_name='tool_call_match',save_filename=True):
     if save_filename:
         save_plot(fig, f'Comparison of {column_name} by server type')
 
-def get_analysis_plots():
-        # Load the CSV file into a DataFrame
-    file_path = './results/metrics.csv'
-    df = pd.read_csv(file_path)
+def get_analysis_plots(metric_file: str):
+    tool_plot = True if "tool" in metric_file else False
```

Ruff (0.11.9)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)